Gruntfile.coffee

```diff
@@ -29,15 +29,6 @@ module.exports = ->
             discover: true
             recursive: true
             silent: true
-          webpack:
```

Comment:
@jonnor the existence of those keys doesn't cause it, but there were some minor config changes in webpack from v1 to v2, so they needed tweaking. But they were also unnecessary, so removed.